ganglia/test_ganglia.py

```diff
@@ -0,0 +1,61 @@
+import unittest
+
+from parameterized import parameterized
+from integration_tests.dataproc_test_case import DataprocTestCase
+
+
+class GangliaTestCase(DataprocTestCase):
+    COMPONENT = 'ganglia'
+    INIT_ACTION = 'gs://dataproc-initialization-actions/ganglia/ganglia.sh'
+    TEST_SCRIPT_FILE_NAME = 'verify_ganglia_running.py'
+
+    def verify_instance(self, name):
+        self.upload_test_file(name)
+        self.__run_command_on_cluster(name,'yes | sudo apt-get install python3-pip')
+        self.__run_command_on_cluster(name,'sudo pip3 install requests-html')
+        self.__run_test_file(name)
+        self.remove_test_script(name)
+
+    def __run_test_file(self, name):
+        cmd = 'gcloud compute ssh {} -- "python3 {} "'.format(
+            name,
+            self.TEST_SCRIPT_FILE_NAME
+        )
+        ret_code, stdout, stderr = self.run_command(cmd)
+        self.assertEqual(ret_code, 0, "Failed to run test file. Error: {}".format(stderr))
+
+    def __run_command_on_cluster(self, name, command):
```

Comment:
Nit: This seems useful enough to go in `DataprocTestCase`

Reply:
yes but unfortunately DataprocTestCase is not uploaded to cluster so I can't apply re-usable solution for this. But I think the same about it and helper methods like this would be nice in the future.

Reply:
Ah gotcha -- after all the tests are merged, we should think about refactoring these utilities into a common library. Pyspark jobs do let you bring additional libraries, fwiw.

But for now I'll go ahead and merge this PR.